core/server/api_container/server/startosis_engine/kurtosis_types/service_config/image_build_spec.go

```diff
@@ -221,10 +220,6 @@ func getOnDiskImageBuildSpecPaths(
 	locatorOfModuleInWhichThisBuiltInIsBeingCalled string,
 	packageContentProvider startosis_packages.PackageContentProvider,
 	packageReplaceOptions map[string]string) (string, string, *startosis_errors.InterpretationError) {
-	if packageId == startosis_constants.PackageIdPlaceholderForStandaloneScript {
```

Comment:
I needed this check so that docker compose packages being called using `import_module` from standalone scripts worked. The error thats returned now when using an `ImageBuildSpec` in a standalone script is a more general relative URL err:

```
Source module locator 'DEFAULT_PACKAGE_ID_FOR_SCRIPT' isn't a valid locator; relative URLs don't work with standalone scripts
```

I think this is valid and still enough to indicate to the user they need to create a package for an ImageBuildSpec to work.